lib/elixir/lib/file/stream.ex

```diff
@@ -73,7 +73,8 @@ defmodule File.Stream do
       start_fun =
         fn ->
           case :file.open(path, read_modes(modes)) do
-            {:ok, device}    -> device
+            {:ok, device}    ->
```

Comment:
Since you are changing this line, can you please remove the indentation of the `->`? We no longer do such alignments in Elixir.